cypress/e2e/plugins/index-management-dashboards-plugin/snapshots_spec.js

```diff
@@ -71,17 +71,20 @@ describe("Snapshots", () => {
 
       // Select all indexes to be included
       cy.get(`[data-test-subj="indicesComboBoxInput"]`).type("test_index_1{enter}");
+      cy.wait(1000);
       cy.get(`[data-test-subj="indicesComboBoxInput"]`).type("test_index_2{enter}");
+      cy.wait(1000);
       cy.get(`[data-test-subj="indicesComboBoxInput"]`).type("test_index_3{enter}");
+      cy.wait(1000);
 
       // if a toast message pops up then dismiss it
       cy.dismissToast();
 
       // Click 'Add' button to create snapshot
-      cy.get(`[data-test-subj="flyout-footer-action-button"]`).contains("Add", { timeout: 3000 }).click({ force: true });
+      cy.get(`[data-test-subj="flyout-footer-action-button"]`).contains("Add", { timeout: 3000 }).click();
 
       // check for success status and snapshot name
-      cy.get("button").contains("Refresh").click({ force: true });
+      cy.get("button").contains("Refresh").click();
```

Comment:
suggestion: node and cypress got faster. so a suggestion is to slow it down a little bit with `should("be.visible")`

for example `cy.get("button").contains("Refresh").should(be.visible).click();`